source/imports/domain/banking-domain-module.js

```diff
@@ -0,0 +1,15 @@
+import BankAccountRouter from '../es-routers/bank-account-router';
+
+const BankingDomain = Space.Module.define('BankingDomain', {
+
+  requiredModules: [
+    'Space.eventSourcing'
+  ],
+
+  singletons: [
+    BankAccountRouter
```

Comment:
Yes, I will change that. That is in line with our latest discussions. I just went with "standard" way of doing things until i have tests running. Thanks. :+1: